config.h.meson

```diff
@@ -0,0 +1,158 @@
+/* jansson_private_config.h.in.  Generated from configure.ac by autoheader.  */
```

Comment:
Have you used some script to generate it? It would be nice to add it as a header.

Reply:
Again I followed the  `ac_converter.py` instructions. Copying the config.h.in file and replacing `#undef` with `#mesondefine`.  `jansson_private_config.h.in` is included in the distributed tarball - I did not generate it myself but the above line is present in the original file as well.